float32 mean_squared_error
float32 y_desired
float32 y_sensed
float32 wave
int32 numoftrial